src/installer/pkg/packaging/rpm/package.targets

```diff
@@ -315,84 +318,31 @@
-      <ConfigJsonName>dotnet-runtime-deps-rpm_config_$(OutputRid).json</ConfigJsonName>
-      <ConfigJsonFile>$(rpmPackagingConfigPath)$(ConfigJsonName)</ConfigJsonFile>
-      <RpmIntermediatesDir>$(PackagesIntermediateDir)$(RpmPackageName)/$(RpmPackageVersion)</RpmIntermediatesDir>
-    </PropertyGroup>
+  <Target Name="GenerateRuntimeDependenciesRpms">
 
     <PropertyGroup>
-      <RpmLayoutDirectory>$(RpmIntermediatesDir)/rpmLayoutDirectory/</RpmLayoutDirectory>
-      <RpmLayoutPackageRoot>$(RpmLayoutDirectory)package_root</RpmLayoutPackageRoot>
-      <RpmLayoutDocs>$(RpmLayoutDirectory)docs</RpmLayoutDocs>
-      <RpmLayoutTemplates>$(RpmLayoutDirectory)templates</RpmLayoutTemplates> <!-- Copyright, Changelog -->
+      <SharedProperties>rpmPackagingConfigPath=$(rpmPackagingConfigPath);DotnetRuntimeDependenciesPackageInstallerFile=$(DotnetRuntimeDependenciesPackageInstallerFile);OutputRid=$(OutputRid);RuntimeDependenciesRpmPkgName=$(RuntimeDependenciesRpmPkgName);RuntimeDepsRpmPackageVersion=$(RuntimeDepsRpmPackageVersion);RuntimeDepsRpmPackageRelease=$(RuntimeDepsRpmPackageRelease);TemplatesDir=$(TemplatesDir)</SharedProperties>
+      <DepsConfigFilePrefix>dotnet-runtime-deps-rpm_config_</DepsConfigFilePrefix>
     </PropertyGroup>
 
-    <!-- Error out if the configuration file doesn't exist -->
-    <Error Condition="!Exists('$(ConfigJsonFile)')" Text="Config file does not exist : '$(ConfigJsonFile)'" />
-
-    <RemoveDir Condition="Exists('$(RpmIntermediatesDir)')" Directories="$(RpmIntermediatesDir)" />
-    <MakeDir Directories="$(RpmIntermediatesDir)" />
-
-    <!-- Create empty rpm layout -->
-    <RemoveDir Condition="Exists('$(RpmLayoutDirectory)')" Directories="$(RpmLayoutDirectory)" />
-    <MakeDir Directories="$(RpmLayoutDirectory)" />
-    <MakeDir Directories="$(RpmLayoutPackageRoot)" />
-    <MakeDir Directories="$(RpmLayoutDocs)" />
-    <MakeDir Directories="$(RpmLayoutTemplates)" />
-    
-     <!-- Copy files to rpm layout -->
-    <ItemGroup>
-         <RDTemplatesFiles Include="$(TemplatesDir)/**/*" />
-    </ItemGroup>
-
-    <Copy SourceFiles="@(RDTemplatesFiles)" DestinationFiles="@(RDTemplatesFiles->'$(RpmLayoutTemplates)/%(RecursiveDir)%(Filename)%(Extension)')" />
-
-    <!-- Replace config json variables -->
-    <ItemGroup>
-      <RuntimeDependenciesTokenValue Include="%RUNTIME_DEPS_RPM_PACKAGE_NAME%">
-        <ReplacementString>$(RuntimeDependenciesRpmPkgName)</ReplacementString>
-      </RuntimeDependenciesTokenValue>
-      <RuntimeDependenciesTokenValue Include="%RUNTIME_DEPS_VERSION%">
-        <ReplacementString>$(RuntimeDepsRpmPackageVersion)</ReplacementString> 
-      </RuntimeDependenciesTokenValue>
-      <RuntimeDependenciesTokenValue Include="%RUNTIME_DEPS_REVISION%">
-        <ReplacementString>$(RuntimeDepsRpmPackageRelease)</ReplacementString>
-      </RuntimeDependenciesTokenValue>
+    <!-- non-portable build - add OutputRid to new item list -->
```

Comment:
I'd prefer describing the purpose here (and similar in below comments):
```suggestion
    <!-- non-portable build - build a single dotnet-runtime-deps package using OutputRid -->
```

Reply:
These were meant to be to-do comments and removed - but I'll update them. 👍